packages/devtools_app/test/extensions/extension_settings_test.dart

```diff
@@ -0,0 +1,215 @@
+// Copyright 2023 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:devtools_app/devtools_app.dart';
+import 'package:devtools_app/src/extensions/extension_settings.dart';
```

Comment:
Should this be exported from `devtools_app` instead?

Reply:
It doesn't necessarily need to be. We haven't had a strong convention around this other than we've included everything in devtools_app.dart that needs to be imported from another package (g3 custom tooling or devtools_test).